sysutils/flash/Portfile

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+PortGroup       github 1.0
+
+github.setup    hypriot flash 2.3.0
+
+platforms       darwin
+categories      sysutils
+license         MIT
+maintainers     @jrjsmrtn openmaintainer
+description     Command line script to flash SD card images of any kind
+long_description    ${description}
+
+checksums       rmd160  d0f45b33da3b557cbf9253a835678310ad9ed27a \
+                sha256  7a31c2fa6a2169633c5d77b205ead4562b110e1c106b3ac4fa72ee55350e0719 \
+                size    16744
+
+depends_run     port:unzip
+
+variant curl description {If you want to flash directly with an HTTP URL} {
+    depends_run-append port:curl
+}
+
+variant aws description {If you want to flash directly from an AWS S3 bucket} {
+    depends_run-append port:py-awscli
+}
+
+variant pv description {To see a progress bar while flashing with the `dd` command} {
+    depends_run-append port:pv
+}
+
+default_variants    +curl +pv
```

Comment:
I'm not a fan of any of these variants. If the user had the ports installed already, then this port would use them even though the variants are not selected. I think you should scrap the variants and just add the dependencies.

Reply:
My reasoning was:

1. _none_ of those variants are required by `flash`;
2. `py37-awscli` has a _lot_ of dependencies and quite frankly, I _never_ had to use AWS downloads since I use that tool;
3. `flash` was created by the HypriotOS team and their images are downloadable directly from their Web site, thus `curl` is a more useful option;
4. `pv` is a _nice-to-have_;
5. therefore I put _sensible_ variants as defaults, giving the user the possibility to not install the bells and whistles.

What do you think?